components/Carousel/Carousel.js

```diff
@@ -1,6 +1,106 @@
 class Carousel {
+  constructor(element) {
+    this.element = element;
+    this.carousel = document.querySelectorAll('.Carousel__item');
```

Comment:
line 4 and 5 could have been combines into a single expression. Nothing wrong with your methods just a suggestion.